TezosKit/Dexter/DexterExchangeClient.swift

```diff
@@ -264,4 +299,51 @@ public class DexterExchangeClient {
       completion: completion
     )
   }
+
+  /// Create an operation to buy Tez with tokens.
+  ///
+  /// - Parameters:
+  ///   - source: The address making the trade.
+  ///   - destination: The destination for the tokens.
+  ///   - signatureProvider: An opaque object that can sign the transaction.
+  ///   - tokensToSell: The number of tokens to sell.
+  ///   - minTezToBuy: The minimum number of Tez to buy.
+  ///   - deadline: A deadline for the transaction to occur by.
+  ///   - completion: A completion block which will be called with the result hash, if successful.
+  public func tradeTokenForTezOperation(
```

Comment:
Is there any reason we wouldn't want to have the actual execution methods call this function to get the operation?

Reply:
I thought there was a difference, and I wanted to avoid picking that up now, but I looked again and there wasn't. I modified the functions to use the Operation functions instead